user/security-in-qubes/mfa.md

```diff
@@ -184,29 +188,34 @@ website](https://docs.nitrokey.com/software/nitropy/all-platforms/installation).
             Proper packaging and distribution for Debian and perhaps Fedora is
             also planned for the mid-long term.*
             **Installing packages using pip or pipx is not recommended!**
-   
+
+
    **both**
 
+
    Shut down your template. Then, either reboot your USB VM (so changes inside
    the template take effect in your USB app qube) or install the packages inside
    your USB VM as well if you would like to avoid rebooting it.
 
-2. Install [qubes-app-yubikey](https://github.com/QubesOS/qubes-app-yubikey) in
+1. Install [qubes-app-yubikey](https://github.com/QubesOS/qubes-app-yubikey) in
```

Comment:
I think this is supposed to be continuation of the list, see https://www.qubes-os.org/doc/mfa/

Reply:
That was not my intention. I can correct that. Thanks for pointing out!